module.json

```diff
@@ -44,6 +44,9 @@
     },
     "arm-ssg": {
       "mbed-hal-arm-ssg": "^1.0.0"
+    },
+    "atmel": {
+      "mbed-hal-atmel": "^0.1.0"
```

Comment:
@0xc0170  
I hope  ^ will allow me to upgrade version 0.X.X.
On other hand ~ will allow only 0.1.X right ?